### PR TITLE
fix(StatusDialogFooter/StatusModalFooter): Prevent footer's buttons overflow

### PR DESCRIPTION
**Pull request by micieslak**
### What does the PR do

When there is no room for footer buttons in their full width, they are shrank and elided instead of going outside the popup.

Closes: #11449

### Affected areas
`StatusDialogFooter`, `StatusModalFooter`

### Screenshot of functionality (including design for comparison)

[Kazam_screencast_00091.webm](https://github.com/status-im/status-desktop/assets/20650004/84e0df31-af9f-4daf-9770-2873eb7df10d)

### CODE REVIEW

File: ui/app/AppLayouts/Communities/popups/FinaliseOwnershipPopup.qml [hunk 1]
```diff
@@ -149,7 +150,11 @@ StatusDialog {
             }
         }
         leftButtons: ObjectModel {
-            StatusBackButton { id: backButton }
+            StatusBackButton {
```
Comment (caybro) [inline ui/app/AppLayouts/Communities/popups/FinaliseOwnershipPopup.qml:153]: Is it just this problematic popup, or do we have a similar problem somewhere else too? (we do have a back button in other places too)

**Review by caybro — APPROVED** [review]
Nice work!

File: ui/app/AppLayouts/Communities/popups/FinaliseOwnershipPopup.qml (hunk 1 above)
Comment (caybro) [reply ui/app/AppLayouts/Communities/popups/FinaliseOwnershipPopup.qml:153]: Cool! Pls also make sure to squash the 2 commits

**Review by alexjba — APPROVED** [review]
Nice!

**Pull request merged**